src/pstress.cpp

```diff
@@ -20,6 +20,9 @@
 #include <mysql.h>
 #include <thread>
 
+/* Global variable to hold pstress source directory */
+std::string fullpath;
```

Comment:
Rename the variable to something like `binary_fullpath`.

Reply:
fixed

Reply:
fixed